src/wui/warehousewindow.cc

```diff
@@ -128,10 +129,11 @@ WarehouseWaresPanel::WarehouseWaresPanel(UI::Panel* parent,
      display_(this, width, wh_, type_, can_act_) {
 	add(&display_, Resizing::kFullSize);
 
+	UI::Box* buttons = new UI::Box(this, 0, 0, UI::Box::Horizontal);
+	add(buttons, UI::Box::Resizing::kFullSize);
+	UI::Button* b;
+
 	if (can_act_) {
```

Comment:
If we now surpass the can_act_ check does this mean spectators can now change stock policies?